mitxgraders/listgrader.py

```diff
@@ -761,9 +758,25 @@ def check_response(self, answer, student_input, **kwargs):
                                            n_expect=len(answers),
                                            partial_credit=self.config['partial_credit'])
 
-        # If the result is ok, display the message for this answer
-        if result['ok'] is True and msg != '':
-            result['msg'] = msg if result['msg'] == '' else result['msg'] + '\n' + msg
+        # Check if all inputs were awarded credit
+        all_awarded = False
+        if not isinstance(self.config['subgrader'], SingleListGrader):
+            # Check to see if all items were awarded credit
+            if all(item['grade_decimal'] > 0 for item in input_list):
```

Comment:
Minor comment: here and below could just do 

`all_awarded=all(item['grade_decimal'] > 0 for item in input_list)`

I believe no need for conditional. Also no need for initialization to False above.

Reply:
Good point - fixed.